### ISSUE TITLE
Merge paper-item-body into paper-item

### ISSUE THREAD

**Issue by JelmarBosscher** [PolymerElements/paper-item#77]
When using the Polymer Starter Kit I followed the demo on the element catalog page.
https://elements.polymer-project.org/elements/paper-item

I assumed that `paper-item-body` is part of the paper-item element.
Perhaps this was a wrong assumption but I found it odd that `paper-item-body` had to be imported separately.

Same goes for the paper-item-icon, but this is already an issue. 
https://github.com/PolymerElements/paper-item/issues/44

**Comment by bicknellr** [comment]
Yeah, this is pretty strange. We've got to work out a better deprecation strategy before we get to this but it's good to have it here so we don't forget. Thanks for filing!

**Comment by simonfuhrmann** [comment]
Why would `paper-item` not simply import `paper-item-body`?
